.gitignore

```diff
@@ -26,39 +10,16 @@ datatracker.sublime-workspace
 /.Python
 /.settings
 /.tmp
-/attic
-/bin
-/bootstrap/node_modules
 /data
 /docker/docker-compose.extend-custom.yml
-/env
-/etc
 /ghostdriver.log
-/htmlcov
```

Comment:
This should come back. It's the output of `coverage html` for when you want to look at the coverage results locally.